### ISSUE TITLE
client crashes when artist's name contains unicode

### ISSUE THREAD

**Issue by klntsky** [mopidy/mopidy-soundcloud#103]
Steps to reproduce:

1. follow [this artist](https://soundcloud.com/countinstack)
2. go to soundcloud/following

I've checked it with two different MPD clients, so I suppose it should be reported here.

**Comment by 8084** [comment]
Looks like a parsing issue somewhere.
Iris shows this instead of the artist's name:

![this](https://user-images.githubusercontent.com/18447310/50492643-e4331280-0a29-11e9-83fe-76d5ddac290d.jpg)

**Comment by 8084** [comment]
Some more links to follow for those who try to resolve this issue:

 [One](https://github.com/mopidy/mopidy/blob/develop/mopidy/models/fields.py#L70):

```python
    def __init__(self, default=None):
        # TODO: normalize to unicode?
        # TODO: only allow unicode?
        # TODO: disallow empty strings?
```

[Two](https://github.com/mopidy/mopidy/blob/f267e8db206df1a9e5c7967af1d1733ab9ce4290/mopidy/compat.py#L33):

```python
 string_types = basestring # noqa
```

From these I conclude that passing artist names exactly as returned by soundcloud API is absolutely OK, and all possible issues are either on `mopidy.models`' side or on the side of a particular client.

**Comment by jodal** [comment]
I agree with the conclusion here. Fixed by #112.
